public_html/extensions/fast_checkout/storefront/view/default/template/responses/checkout/payment.tpl

```diff
@@ -180,7 +180,7 @@ $guest_data = $this->session->data['fc']['guest'];
                                                 <td style="vertical-align: middle;">
                                                     <label for="<?php
                                                             $idd = str_replace('.', '', $quote['id']);
-                                                                echo preg_replace('/[^a-zA-Z0-9\.-_]/', '', $idd.$quote['id']); ?>"
+                                                                echo preg_replace('/[^a-zA-Z0-9\.-_]/', '', $idd.$quote['id']); ?>_title"
```

Comment:
please clarify what is for

Reply:
One of our shipper extension changed the shipping method title via ajax. The extra "_title" was used so i can target the correct label via javascript.

Reply:
i think this changes broke labeling. You changed attribute "for" of LABEL tag but not changed attribute "id" of INPUT tag (see code below). When you click on shipping icon it WILL NOT select radiobutton. Can you check? Just set some icon for shipping extension to see

Reply:
Yes i can check it. i will do that along with testing the update data you suggested just now.

Reply:
I tried adding a shipping icon but the icon was not shown in the page.